### ISSUE TITLE
container width/height get 0 when visible is false in v4.

### ISSUE THREAD

**Issue by humanhighway** [pixijs/pixi.js#2893]
as title, works normal in v3.
reproduce code:

```
      var stage = new PIXI.Container();
      var renderer = PIXI.autoDetectRenderer(500, 500);
      document.body.appendChild(renderer.view);

      var container = new PIXI.Container();
      stage.addChild(container);
      var g = new PIXI.Graphics();
      g.beginFill(0xff0000);
      g.drawRect(0, 0, 300, 200);
      g.endFill();
      container.addChild(g);
      renderer.render(stage);

      console.log(this.container.width, this.container.height);
      console.log();
      container.visible = false;
      console.log(this.container.width, this.container.height);

      var render = function(){
        requestAnimationFrame(render);
        renderer.render(stage);
      };
      render();
```

**Comment by GoodBoyDigital** [comment]
Thanks for sharing @humanhighway 👍

**Comment by englercj** [comment]
Should be fixed by #2956 which has been merged into the latest dev.

You can see how to download the latest build [in the FAQs](https://github.com/pixijs/pixi.js/wiki/FAQs#where-can-i-get-a-build).

Thanks!
